graphql/schema/types/permissions.graphql

```diff
@@ -12,6 +12,7 @@ type DistroPermissions {
 type Permissions {
   canCreateDistro: Boolean!
   canCreateProject: Boolean!
+  canUpdateAdminSettings: Boolean!
```

Comment:
I don't feel too strongly because this pattern is used here but I feel isAdmin is a better name for this.

Reply:
+1 I also like `isAdmin` 🤔 at the very least, I think I'd prefer using the [same verbiage](https://github.com/evergreen-ci/evergreen/blob/4d5228b8e7e87d06b5ff55b0e5fcf8c663069a0f/globals.go#L1090-L1093) from the permission levels like `canEditAdminSettings`

Reply:
I think using `isAdmin` would be fine. Imo there's no comparable equivalent for the other fields (`isAbleToCreateProject` is awk, `isProjectCreator` is not ideal either) so breaking the pattern makes sense.